vaadin-icons-flow-parent/vaadin-icons-flow-integration-tests/src/test/java/com/vaadin/flow/component/icon/tests/IconIT.java

```diff
@@ -15,6 +15,8 @@
  */
 package com.vaadin.flow.component.icon.tests;
 
+import static org.junit.Assert.assertTrue;
```

Comment:
This import might be not necessary as `Assert` is imported at the [22nd line](https://github.com/vaadin/flow-components/pull/1810/files#diff-2f2b6ccf817dada8aa49e68fd61ade90d36c2a1bde5a0f76e6e0000ac1aeeb2aR22).

Reply:
Done.